package.json

```diff
@@ -31,12 +31,12 @@
     "es6-shim": "0.35.5",
     "express": "^4.17.1",
     "getmac": "^1.4.6",
-    "json-schema": "^0.2.3",
+    "json-schema": "^0.2.5",
     "json-socket": "^0.3.0",
     "mqtt": "^3.0.0",
     "uuid": "^3.3.2",
     "winston": "^3.2.1",
-    "@open-iot-service-platform/oisp-sdk-js": "^1.1.0"
+    "@open-iot-service-platform/oisp-sdk-js": "https://github.com/Open-IoT-Service-Platform/oisp-sdk-js.git#a8d791b7d1bcc030a9e781cb69038ccbda5877e1"
```

Comment:
I don't want to get into this sort of dependency disaster. I will do an SDK release which we can base this PR on.

Reply:
OISP JS SDK v1.1.1 published.